include/deal.II/matrix_free/matrix_free.h

```diff
@@ -640,6 +640,13 @@ class MatrixFree : public Subscriptor
    */
   //@{
   /**
+   * Returns whether a given FiniteElement @p fe is supported by this class.
+   */
+  template <int spacedim>
+  static
+  bool supports_fe (const FiniteElement<dim, spacedim> &fe);
```

Comment:
I was asking myself who is supporting whom here. What do you think of the name `is_supported` instead? The prefix `is_` is always associated with a true/false question so I think that would match here.